_infra/helm/frontstage/Chart.yaml

```
@@ -18,5 +18,5 @@ version: 1.2.1
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
-appVersion: 2.2.99
 
+appVersion: 2.2.100
```

```suggestion
appVersion: 2.3.0
```

This isn't your fault or anything, I just think if we're getting to patch number 100 it's probably time to roll that sucker over 🤣